src/gui-source/Kolibri/Kolibri.cpp

```diff
@@ -109,7 +109,7 @@ void kolibriScriptPath(char *buffer, const DWORD MAX_SIZE)
 }
 
 void kolibriFailedMessage() {
-	MessageBox(HWND_DESKTOP, concatenateWchar(getStr(ID_STRING_21_en), getWC(getKolibriHomePath()), getWC("\\logs\\kolibri.log")), L"Kolibri", MB_OK | MB_ICONINFORMATION);
+	MessageBox(HWND_DESKTOP, concatenateWchar(getStr(ID_STRING_21_en), getWC(getKolibriHomePath()), getWC("\\logs")), L"Kolibri", MB_OK | MB_ICONINFORMATION);

```

Comment:
Maybe add a back-slash at the end to denote it's a directory?  Like `...\logs\` instead?